internal/action/setup.go

```diff
@@ -184,11 +201,14 @@ func (s *Action) initHasUseablePrivateKeys(ctx context.Context, crypto backend.C
 	// only list "usable" / properly trused and signed GPG keys by requesting
 	// always trust is false. Ignored for other backends. See
 	// https://www.gnupg.org/gph/en/manual/r1554.html
-	kl, err := crypto.ListIdentities(gpg.WithAlwaysTrust(ctx, false))
```

Comment:
What about `withAlwaysTrust` and the comment above?
We would still need it for gpg, no?

Reply:
Yes, and it's still being set but at the very beginning of the setup method. I'll need to move the comment (and do some more tests with GPG later).